.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: 🚀 Feature Request
-description: Suggest an idea for this project
+about: Suggest an idea for this project
 title: "[feat.]: <title>"
```

Comment:
It is recommended to maintain consistent style for these types: bug, feature, enhancement and question;
current is:
1. bug has no title
2. enhancement/question has a upper and full title as 'Enhancement/Question'
3. feature has a lower and short title as 'feat.'